samples/cpp/benchmark_app/main.cpp

```diff
@@ -552,7 +623,14 @@ int main(int argc, char* argv[]) {
             next_step();
             slog::info << "Skipping the step for loading network from file" << slog::endl;
             auto startTime = Time::now();
-            compiledModel = core.compile_model(FLAGS_m, device_name);
+            ov::AnyMap properties = {};
+            if (if_auto || if_multi) {
+                std::string virtual_device = if_auto ? "AUTO" : "MULTI";
+                if (if_auto && if_multi)
+                    virtual_device = split(device_name, ':').at(0);
```

Comment:
Should it be just
```suggestion
                virtual_device = split(device_name, ':').at(0);
```
? Similar to how you removed `std::string virtual_device = if_auto ? "AUTO" : "MULTI";` at line 350